doc: edit agl-vms.rst adjust to v1.3 (#4071)
projectacrn/acrn-hypervisor: doc/tutorials/agl-vms.rst

entertainment (RSE).  For the software, there are three VMs running on
top of ACRN:

> * Clear Linux OS runs as the service OS (Service VM) to control the cluster meter,

Reviewer: ```suggestion
* Clear Linux OS runs as the Service VM to control the instrument cluster,
```

> * an AGL instance runs as a user OS (User VM) controlling the IVI display, and

Reviewer: ```suggestion
* an AGL instance runs as a User VM controlling the In-Vehicle Infotainment (IVI) display, and
```

> * a second AGL User VM controls the RSE display.

Reviewer: ```suggestion
* a second AGL User VM controls the Rear Seat Entertainment (RSE) display.
```

      # swupd autoupdate --disable

#. This demo setup uses a specific release version (31080) of Clear
   Linux OS which has been verified to work with ACRN. In case you
   unintentionally update or change the Clear Linux OS version, you can
   fix it again using::

      # swupd verify --fix --picky -m 31080

#. Use `acrn_quick_setup.sh <https://github.com/projectacrn/acrn-hypervisor/blob/84c2b8819f479c5e6f4641490ff4bf6004f112d1/doc/getting-started/acrn_quick_setup.sh>`_ 

>    to automatic installation ACRN::

Reviewer 2: ```suggestion
   to automatically install ACRN::
```

>       # sh acrn_quick_setup.sh -s 31080 -i 

Reviewer: ```suggestion
      $ sudo ./acrn_quick_setup.sh -s 31080 -i 
```

> #. After installation, the system will start up again automatically

Reviewer 2: ```suggestion
#. After installation, the system will automatically start.
```

#. Reboot the system, choose "ACRN Hypervisor" and launch Clear Linux OS

>    SOS. If the EFI boot order is not right, use :kbd:`F10`

Reviewer 2: Please remove this instance and all possible instances of SOS and UOS and replace with Service VM / User VM.

      $ cd workspace
      $ git clone https://github.com/projectacrn/acrn-kernel
      $ git checkout tags/acrn-2019w39.1-140000p
      $ cp kernel_config_uos .config
      $ make olddefconfig

>       $ vi .config

Reviewer: Shouldn't this `.config` editing step be done *before* running `make olddefconfig`?

Author: Yes, you are right, I will make changes

   the following content::

      #!/bin/bash
      set -x

      offline_path="/sys/class/vhm/acrn_vhm"

      # Check the device file of /dev/acrn_hsm to determine the offline_path
      if [ -e "/dev/acrn_hsm" ]; then
      offline_path="/sys/class/acrn/acrn_hsm"
      fi

      function launch_clear()
      {
      mac=$(cat /sys/class/net/e*/address)
      vm_name=vm$1
      mac_seed=${mac:9:8}-${vm_name}

      #check if the vm is running or not
      vm_ps=$(pgrep -a -f acrn-dm)
      result=$(echo $vm_ps | grep -w "${vm_name}")
      if [[ "$result" != "" ]]; then
        echo "$vm_name is running, can't create twice!"
        exit
      fi

      #logger_setting, format: logger_name,level; like following
      logger_setting="--logger_setting console,level=4;kmsg,level=3"

      #for memsize setting
      mem_size=2048M

>       acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge \

Reviewer: Just double-checking: do we still have the `-c` option in that version of `acrn-dm`?

Author: yes, it's still have in that version

                        sleep 1
                        echo 0 > $i/online
                        online=`cat $i/online`
                      done
                      echo $idx > ${offline_path}/offline_cpu
              fi
      done

      launch_clear 1 1 "64 448 8" 0x000F00 agl

#. Create the ``launch_rse.sh`` script for the AGL RSE VM with this content::

      #!/bin/bash

      set -x 

      offline_path="/sys/class/vhm/acrn_vhm"

      # Check the device file of /dev/acrn_hsm to determine the offline_path
      if [ -e "/dev/acrn_hsm" ]; then
      offline_path="/sys/class/acrn/acrn_hsm"
      fi

      function launch_clear()
      {
      mac=$(cat /sys/class/net/e*/address)
      vm_name=vm$1
      mac_seed=${mac:9:8}-${vm_name}

      #check if the vm is running or not
      vm_ps=$(pgrep -a -f acrn-dm)
      result=$(echo $vm_ps | grep -w "${vm_name}")
      if [[ "$result" != "" ]]; then
        echo "$vm_name is running, can't create twice!"
        exit
      fi

      #logger_setting, format: logger_name,level; like following
      logger_setting="--logger_setting console,level=4;kmsg,level=3"

      #for memsize setting
      mem_size=2048M

>       acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge -U 495ae2e5-2603-4d64-af76-d4bc5a8ec0e5 \

Reviewer: Same question re: `-c` parameter

Author: yes
